README.md

```diff
@@ -1 +1,16 @@
-# SQL-
```

Comment:
###  **Update** from Multiple Tables
* UPDATE table_name1 SET table_name1.col1= table2.col1  from table2 left join table3  on table2.id=table3.id where table2.name in (select name from table3)